tests/unit/create-plugin.test.js

```diff
@@ -63,16 +63,12 @@ tap.test('createPlugin edge cases', (t) => {
     t.end()
   })
 
-  t.test('should not set operation name to unknown when document is null', (t) => {
+  t.test('should not set operation name when document is null', (t) => {
     const responseContext = {}
     const hooks = createPlugin(instrumentationApi)
     const operationHooks = hooks.requestDidStart({})
     operationHooks.willSendResponse(responseContext)
-    t.equal(
-      operationSegment.name,
-      'GraphQL/operation/ApolloServer/<unknown>',
-      'should set operation to unknown'
-    )
+    t.equal(operationSegment.name, undefined, 'should set operation to unknown')
```

Comment:
This seems wrong. We should never have a segment with no name at all.

Reply:
it's a unit test where `createSegment` is stubbed and where it would've named it accordingly

Reply:
oh i see, this is specifically testing this hook and this hook doesn't touch the segment name anymore.

sorry, i was getting this and the previous test change merged in my mind.

Reply:
i might recommend a `validationDidStart` equivalent to what was previously here (verifying named to unknown) but our versioned testing is so in-depth i know that is covered. so not critical.

Reply:
yea I can do that. but it won't update it to unknown, it'll also be `undefined`.

Reply:
actually. I'm going to change this test and add a different one that is asserting the default behavior of the segment name